src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        System.out.println(Lottery.getRandomBall());
+        System.out.println(Lottery.getRandomBall());
+        System.out.println(Lottery.getRandomBall());
```

Comment:
You're creating several instances of the Lottery class implicitly by calling the static method getRandomBall multiple times. According to the checklist, it's better to create an instance of the class and then call the method on that instance.